readme.md

```diff
@@ -38,13 +38,12 @@ An object to transform into snake case (keys only).
 *Optional*  
 Type: `object`
 
-Optional settings for the conversion.  Included options are:
+###### deep
 
-* deep
-    * Type: `boolean`
-    * Default: `true`
-    * If set to false, only the top-level keys will get converted.
-    * Example: `snakeCaseKeys({topLevel: { childLevel: 'not converted }}, { deep: false })` => `{ top_level: { childLevel: 'not converted}}`
+Type: `boolean`
```

Comment:
Add a trailing space after boolean`, otherwise it will be rendered without the linebreak. You can preview Markdown from a GH diff like this:

![](https://d.pr/i/is97MU+)